scipy/linalg/flapack.pyf.src

```diff
@@ -735,11 +763,296 @@ interface
           :: lwork=2*minmn+MAX(maxmn,nrhs)
           ! check(lwork>=2*minmn+MAX(maxmn,nrhs))
```

Comment:
Hi, All

Reimplementing everything in Cython would require much more time from me, because I have almost no experience with it. On the other hand, these wrappers are ready and 'linalg.lstsq' is modified accordingly in PR #4491. So, users can immediately use faster least-squares solvers.